spec/controllers/callbacks_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CallbacksController, type: :controller do
+
+end
```

Comment:
@ruishine a part of setting up fb auth is putting things into CallBackscontroller (https://github.com/plataformatec/devise/wiki/OmniAuth:-Overview). The controller was probably generated via terminal (rails generate controller....) and the test was automatically created.. To your question: I won't tell you not to add tests 😉 , on the other hand it could be a bit tricky... here are some thoughts: https://gist.github.com/jittuu/792715